client_src/js/lib/modules/printerSettings.js

```diff
@@ -1424,6 +1428,3 @@ class PrinterSettings {
     }
   }
 }
-module.exports = {
-  updatePrinterSettingsModal
-};
```

Comment:
I use this export in printer manager

Reply:
URL?

Reply:
https://github.com/OctoFarm/OctoFarm/blob/55a56934613d9fca1208a4743a082f97e926a899/client_src/js/printerManagementRunner.js#L6